centml/compiler/backend.py

```diff
@@ -57,7 +56,10 @@ def module(self):
     def inputs(self):
         self._inputs = None
 
-    def __get_model_id(self, flow_graph: hidet.FlowGraph) -> str:
+    def _get_model_id(self, flow_graph: hidet.FlowGraph) -> str:
```

Comment:
When I originally named with double underscore, I misunderstood how python interpreted "__" in front of functions. I thought it would make the function private but I don't think python does that, and I'm not sure if we want that for all the functions.

The double underscore invokes "name mangling" which means to call the function by the test I have to do something like `self.runner._Runner__get_model_id(...)` which honestly I thought just looked ugly. 

![image](https://github.com/CentML/centml-python-client/assets/96326431/039aa084-3dfe-4afb-88ca-f13f4be0f406)